sycl/include/sycl/detail/pi.h

```diff
@@ -77,9 +77,10 @@
 // 12.22 Add piGetDeviceAndHostTimer to query device wall-clock timestamp
 // 12.23 Added new piextEnqueueDeviceGlobalVariableWrite and
 // piextEnqueueDeviceGlobalVariableRead functions.
+// 13.24 Added a new argument to piextProgramSetSpecializationConstant.
 
-#define _PI_H_VERSION_MAJOR 12
-#define _PI_H_VERSION_MINOR 23
+#define _PI_H_VERSION_MAJOR 13
```

Comment:
We are not allowing ABI breaking changes currently, so please add a new version instead.
+@kbenzie, please note this change to PI API and take it into Unified Runtime

Reply:
Do you mean revert `_PI_H_VERSION_MAJOR` back to `12` and keep the change to `_PI_H_VERSION_MINOR` at `24`?

Reply:
I've created https://github.com/oneapi-src/unified-runtime/issues/269 to track these changes.

Reply:
> Do you mean revert `_PI_H_VERSION_MAJOR` back to `12` and keep the change to `_PI_H_VERSION_MINOR` at `24`?

Yes, but more importantly re-do changes so that existing interfaces are not changed (and add new ones)

Reply:
I'm not sure why we would need the new entry, I thought it was OK to introduce PI API changes (without waiting for the breaking window)?

Reply:
Because PI API crosses libraries boundaries (plugins <-> SYCL RT) and should be stable.
We've been following https://github.com/intel/llvm/blob/sycl/sycl/doc/developer/ABIPolicyGuide.md
I see we need to define there what the next ABI breaking windows is: @stdale-intel ?

Reply:
We've discussed this in the call yesterday and came to a conclusion that PI's API changes do not need to wait for the ABI break window, @smaslov-intel are you OK with bumping the major/minor versions and updating the signature of `piextProgramSetSpecializationConstant`?